calculator.go

```diff
@@ -1,13 +1,72 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"fmt"
+	"math"
+)
+
 // Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+func Add(nums ...float64) float64 {
```

Comment:
This is fine, but it allows us to call `Add` with zero or one arguments, which doesn't really make sense, does it? Of course we could check the number of arguments at runtime, but the compiler can be a good friend (or a powerful enemy). Let's get it on our side, and make it impossible to even _compile_ programs which don't pass at least two arguments to `Add` and co. Can you see how to do that?

Reply:
You've resolved this, so it's really helpful if you can mark it resolved (click 'Resolve conversation'). This saves your reviewer from having to search through the code and check each point individually to see if it's been done! And we appreciate it.